### DISCUSSION TITLE
more examples for tap_action

### DISCUSSION THREAD

**Discussion by cinghialino** [kalkih/mini-graph-card#676] [Q&A]
I can't get it to work the tap_action url thing. Can we get more examples for the tap_action function?
I want to click on the graph and open an url in a different tab

I've used 

```
tap_action: url
url: www.google.com
```

but it doesn't work

**Comment by jlsjonas** [comment]
https://www.home-assistant.io/lovelace/actions/ should be quite helpful 😉
